fix(wfs): use bigger integer data type for indices array

gliegard opened the pull request:
## Description
fix(wfs): use bigger integer data type for indices array

## Motivation and Context
Prior to this change, there was a bug, visible in the wfs globe example :
if you we set buse line WFS layer Level to 10 or less, you will see bugged lines.
The bug comes from the big size of the created LineSegments.


![screenshot-lines-bug](https://user-images.githubusercontent.com/15127065/34168871-f508740c-e4e5-11e7-9a4a-489f50f422fa.jpg)





![screenshot-lines-ok](https://user-images.githubusercontent.com/15127065/34168909-0af4c446-e4e6-11e7-9490-71357073fae7.jpg)

peppsac commented:
Performance wise it's preferable to have a single object than many objects.

Can you elaborate on the original bug? Maybe the vertices count was too big for the indices array (which use 16 bit ushort)?

And btw, if you modify line drawing within WFS it could be a good idea to switch to `MeshLine` (see GpxUtils.js)

gliegard committed:
fix(wfs): use bigger integer data type for indices array, for THREE.LineSegments.
Prior to this change, there was a bug, visible in the wfs globe example :
if you we set buses line WFS layer Level to 10 or less, you will see bugged lines.
The bug comes from the big size of the created LineSegments indices array.

This change use a 32 bit ushort instead of a 16 bit ushort for indices array.

gliegard commented:
You're right, changing 16 bit ushort to 32 bit ushort for indices array solve the problem.
I've removed my changes to do this.

I'll look at MeshLine later.

peppsac reviewed (approved):
You could have used Uint32Array onnly when needed but that doesn't matter much.

Thanks for the fix.
